sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_rules_async.py

```diff
@@ -50,8 +50,9 @@ async def test_async_mgmt_rule_create(self, servicebus_namespace_connection_stri
             "@param": datetime(2020, 7, 5, 11, 12, 13),
         })
 
-        sql_filter = SqlRuleFilter("Priority = @param1", parameters={
+        sql_filter = SqlRuleFilter("Priority = @param1 and Level = @param2", parameters={
```

Comment:
does and vs. AND matter?

Reply:
as far as I know sql syntax should be case-insensitive when working with databases.

but I checked the [service doc](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-sql-filter)
 and found that capital words are used for sql syntax so I will align with the doc.

thanks for the heads up on this!